### ISSUE TITLE
Linux install with GPP - visual problems

### ISSUE THREAD

**Issue by benzin1984** [Kopernicus/Kopernicus#259]
I have some strange visual artifacts on shadow parts of bodies, Ciro (sun), and rings. Running on Ubuntu 16.04 with Ryzen 7 cpu and Nvidia 1050 gpu, KSP 1.3.1 fresh install, minimum addons, and Galileo Planet Pack. Without GPP, no visual artifacts. Below are examples of what I'm seeing:

Ciro: https://i.imgur.com/hakrAFQ.png
I tracked down the config lines causing the artifacts: In GPP/GPP_Planets/Ciro.cfg, section for Coronas cause the corona bugs. In the ScaledVersion section, line "sunspotTex = GPP/GPP_Textures/Ciro.dds" causes the sun's surface bug. I suspect maybe the shader that displays emanating light isn't working properly? Not sure how I can fix.

Tellumo rings: https://i.imgur.com/VPsNfAe.png

All bodies in the shade section show a strange moire pattern: https://i.imgur.com/tp7nV0J.png , outside shade in the lit section bodies look normal.

I tried ModuleManager 2.8.1 and 3.0.1 versions, no difference. Couldn't see any obvious errors in logs. Sorry if it's something obvious that I've missed on the install. I just followed GPP install steps. Thank you..

Edit: running 64 bit, KSP.x86_64. Also, tried to start from this script: LC_ALL=C LD_PRELOAD="libpthread.so.0 libGL.so.1" __GL_THREADED_OPTIMIZATIONS=1 ./KSP.x86_64 -force-glcore -force-gfx-st  
But makes no difference. Also, tried setting GLIgnoreGLSExtReqs=true on NVidia application profile, also makes no difference..

**Comment by kjack1111** [comment]
This is a known issue when you run big packs in 32-bit. Not sure if Linux supports 64-bit, And this may be an entirely unrelated problem. In that case i'm handing it over to more experienced users.

**Comment by Galileo88** [comment]
@benzin1984 copy these textures into your GPP_Textures folder and overwrite when asked. If this doesn't fix the issue, I don't know what the problem could be.

https://www.dropbox.com/s/sfrlo8ooodhqdvs/GPP.zip?dl=1

**Comment by benzin1984** [comment]
Thank you for prompt replies. I've replaced the .dds textures from your file, Galileo, but made no difference. Same problems, sorry..

**Comment by kjack1111** [comment]
The issue we think is occurring that Linux reads textures differently, which is the result of the issue. But that has been disproved, so fun and games are abound!

Unfortunately i must go to sleep, but i wish you all luck.

**Comment by Galileo88** [comment]
Thanks, hopefully we can figure this out soon

**Comment by benzin1984** [comment]
Also thought I'd mention, I was running Nvidia driver 387.34. Just updated to 390.12, no difference. The pc is brand new, gpu is plain 1050 nvidia. I have no visual bugs in plain KSP, or any other 3d programs or games, so far.

**Comment by 7ranceaddic7** [comment]
@kjack1111  

"Not sure if Linux supports 64-bit, And this may be an entirely unrelated problem."

Yes, Linux runs KSP.x86_64; it's all I run.

**Comment by kjack1111** [comment]
Thanks for the information, will keep in mind.

**Comment by benzin1984** [comment]
Galileo, I now think it's a problem with the .dds textures after all. It only seems to affect textures that are copied multiple times by KSP, like the rings, corona, and gradient. So far, I think it's /GPP/terrain/gradient.dds, /GPP/GPP_Textures/Ciro_corona.dds, Ciro.dds and Tellumo_ring.dds. I don't know what compression settings etc I need to use on these .dds texture files to make them load correctly in Linux, I'm still experimenting. Can someone tell me what format and compression algo these ring.dds and corona.dds should be? I tried rgb8, rgba8, abgr8, aexp, with dxt1, dxt5, bct/dxt5, and all sorts of permutations, nothing worked. I made the gradient.dds work with no compression, so at least the lines on planets are gone.

But this is definitely not a Kopernicus issue. Hence, I'm closing this bug report. Sorry if I wasted a dev's time on it. I'll raise an issue under GPP github.
